From 59642c5255461cf8255d96a7cf7b8ae2ea576dc0 Mon Sep 17 00:00:00 2001
From: Kenneth Feng <kenneth.feng@amd.com>
Date: Mon, 11 Mar 2024 14:14:16 +0800
Subject: drm/amd/pm: disable gpo temporarily
Git-commit: d430e856ed009f9a98f12b6d6812d7ee1d7f3e1c
Patch-mainline: v6.11-rc1
References: jsc#PED-9898 jsc#PED-10191 jsc#PED-10197 jsc#PED-10226 jsc#PED-10237 jsc#PED-10340 jsc#PED-10852 jsc#PED-11022

disable gpo temprarily since it is not ready in fw

Signed-off-by: Kenneth Feng <kenneth.feng@amd.com>
Reviewed-by: Jack Gui <Jack.Gui@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Acked-by: Patrik Jakobsson <pjakobsson@suse.de>
---
 drivers/gpu/drm/amd/pm/swsmu/smu14/smu_v14_0_2_ppt.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/amd/pm/swsmu/smu14/smu_v14_0_2_ppt.c b/drivers/gpu/drm/amd/pm/swsmu/smu14/smu_v14_0_2_ppt.c
index 0e27cde82193..b594b748de47 100644
--- a/drivers/gpu/drm/amd/pm/swsmu/smu14/smu_v14_0_2_ppt.c
+++ b/drivers/gpu/drm/amd/pm/swsmu/smu14/smu_v14_0_2_ppt.c
@@ -1867,7 +1867,9 @@ static const struct pptable_funcs smu_v14_0_2_ppt_funcs = {
 	.set_df_cstate = smu_v14_0_2_set_df_cstate,
 	.send_hbm_bad_pages_num = smu_v14_0_2_smu_send_bad_mem_page_num,
 	.send_hbm_bad_channel_flag = smu_v14_0_2_send_bad_mem_channel_flag,
+#if 0
 	.gpo_control = smu_v14_0_gpo_control,
+#endif
 	.get_ecc_info = smu_v14_0_2_get_ecc_info,
 };
 
-- 
2.46.1

